controllers/apps/v1beta3/emqx_handler.go

```diff
@@ -47,42 +47,40 @@ import (
 
 var _ reconcile.Reconciler = &EmqxBrokerReconciler{}
 
+const (
+	EmqxContainerName      = "emqx"
+	ReloaderContainerName  = "reloader"
+	ReloaderContainerImage = "emqx/emqx-operator-reloader:0.0.1"
+	EmqxAnnotationsKey     = "emqxContainerNames"
```

Comment:
I think this one should put in `pkg/handler`, and maybe `ManageContainersAnnotation = apps.emqx.io/manage-containers` is better than `EmqxAnnotationsKey = emqxContainerNames `